lib/signaling/v2/room.js

```diff
@@ -119,6 +119,11 @@ class RoomV2 extends RoomSignaling {
       _trackPrioritySignaling: {
         value: new options.TrackPrioritySignaling(getTrackReceiver, { log }),
       },
+      _trackSubscriptionsSignaling: {
+        value: options.TrackSubscriptionsSignaling
+          ? new options.TrackSubscriptionsSignaling(getTrackReceiver, { log })
+          : null
+      },
```

Comment:
@ceaglest It will be instantiated only for RoomV3, because this code only runs if RoomV2 is acting as a base class for RoomV3. Otherwise, this property is `null`.